src/Symfony/Component/Translation/Dumper/FileDumper.php

```diff
@@ -94,6 +94,20 @@ public function dump(MessageCatalogue $messages, $options = array())
      *
```

Comment:
missing `options` param and I think you should reverse the description for `format` and `formatCatalogue`.